content/08.methods.md

```diff
@@ -9,10 +9,10 @@ We wrote the manuscript in markdown and tracked changes using git.
 Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
 
 To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
-We supported citations to the following identifier types (in order of preference): DOIs, PubMed IDs, arXiv IDs, and URLs.
+We supported citations to the following identifier types (in order of preference): DOIs, PubMed Central IDs, PubMed IDs, arXiv IDs, and URLs.
```

Comment:
Should we remove PubMed IDs now and keep only PubMed Central IDs in this list?

Reply:
I think I'll have to edit the methods section one final time to make sure things are up to date. Let's see at that time whether we have any PubMed IDs references?